pkg/port/models.go

```diff
@@ -193,7 +193,8 @@ type EntityMappings struct {
 }
 
 type Port struct {
-	Entity EntityMappings `json:"entity"`
+	Entity       EntityMappings `json:"entity"`
+	ItemsToParse string         `json:"ItemsToParse"`
```

Comment:
```suggestion
	ItemsToParse string         `json:"itemsToParse"`
```

Reply:
i think we need to add support for yaml in here

Reply:
In general or just for the itemsToParse?
If in general it maybe be better to create separate task for it.

Reply:
its not that big of a deal
 just add yaml:"<theNameInTheYml>" and the name should be same as json

Reply:
Got it, done.

Reply:
we talked about it
yaml

Reply:
Done